Makefile.am

```diff
@@ -23,6 +23,8 @@ EXTRA_DIST = LICENSE ChangeLog.v1 ChangeLog.libgcj			\
 	libffi.xcodeproj/project.pbxproj				\
 	libtool-ldflags
 
+# local.exp is generated by configure
+CLEANFILES = local.exp
```

Comment:
The Automake rule is that if `configure` made it, then `distclean` should remove it; so this should be `DISTCLEANFILES` instead.